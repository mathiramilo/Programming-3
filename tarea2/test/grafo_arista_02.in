leer_mst
4 no-dirigido
1: 2 p3 3 p4 .
2: 4 p5 .
3: .
4: .
0: .
grafo_arista 1 4 1
Fin