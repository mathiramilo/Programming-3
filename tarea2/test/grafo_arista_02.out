1>2>Imprimiendo Grafo
Cantidad de vÃ©rtices: 4
Cantidad de aristas: 3
1: (2,3)(3,4)(4,1).
2: (1,3).
3: (1,4).
4: (1,1).
3>Fin.
